migrations/0012_add_max_processes_column.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE processes ADD COLUMN nproc bigint;
+UPDATE processes SET nproc = 0;
```

Comment:
Will this break existing applications by setting the ulimit in the task definition to 0?